src/styles/components/floating-vue/floating-vue.css

```diff
@@ -0,0 +1,316 @@
+/* originated from node_modules/floating-vue/dist/style.css */
+
+@import '../../tools/z-index.css';
+
+:root {
+  --ec-tooltip-max-width: 260px;
+}
+
+/* stylelint-disable selector-max-class,selector-attribute-quotes */
+
+.v-popper__popper .resize-observer {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: -1;
+  width: 100%;
+  height: 100%;
+  border: none;
+  background-color: transparent;
+  pointer-events: none;
+  display: block;
+  overflow: hidden;
+  opacity: 0;
+}
+
+.v-popper__popper .resize-observer object {
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  overflow: hidden;
+  pointer-events: none;
+  z-index: -1;
+}
+
+.v-popper__popper {
+  /* EC: added */
+  &.ec-tooltip,
+  &.ec-popover {
+    @mixin ec-z-level-all;
+
+    @apply tw-z-tooltip;
+  }
+
+  /* z-index: 10000; EC: removed */
+  top: 0;
+  left: 0;
+
+  &:focus {
+    @apply tw-outline-none; /* EC: added */
+  }
+}
+
+.v-popper__popper.v-popper__popper--hidden {
+  visibility: hidden;
+  opacity: 0;
+  transition: opacity 0.15s, visibility 0.15s;
+  pointer-events: none;
+}
+
+.v-popper__popper.v-popper__popper--shown {
+  visibility: visible;
+  opacity: 1;
+  transition: opacity 0.15s;
+}
+
+.v-popper__popper.v-popper__popper--skip-transition,
+.v-popper__popper.v-popper__popper--skip-transition > .v-popper__wrapper {
+  transition: none !important;
```

Comment:
I guess all these `!important;` properties are needed in order to override the styles that the library applies, right?

Reply:
yes, if it's a non tailwind piece of code, it comes from the original CSS provided by floating-vue. I don't want to touch these or refactor them because I want to keep the amount of changes at minimum so it will be easier to migrate in the future.